verify.rb

```diff
@@ -2,45 +2,67 @@
 require 'yaml'
 require 'fastimage'
 $output=0;
+$image_error_output=0;
+$max_size = 2500 # Max file size (Bytes)
+$max_size_ignore = false #Ignore max file size errors
```

Comment:
This value never changes, so I vote to remove it and the associated conditional on line 21.